internal/common/vote/vote.go

```diff
@@ -0,0 +1,191 @@
+package vote
+
+import (
+	"math/rand"
+	"time"
+)
+
+func Get_votes_for_rule(ruleID string, islandID int) map[int][]int {
+	votes_layout := make(map[int][]int)
+	rand.Seed(time.Now().UTC().UnixNano())
+	i := 1
+	for {
+		votes_layout[i] = []int{rand.Intn(100), rand.Intn(100)}
+		i++
+		if i > 6 {
+			break
+		}
+	}
+	return votes_layout
+}
+
+func VoteRule(ruleID string, islandID int) (int, int, bool) {
+
+	//Get votes from islands for the rule changes.
+	votes_layout := make(map[int][]int)
+	votes_layout = Get_votes_for_rule(ruleID, islandID)
+
+	//Calculate results of each island.
+	results_of_all_islands := make(map[int][]int)
+	j := 1
+	for {
+		if votes_layout[j][0] > votes_layout[j][1] {
+			results_of_all_islands[j] = []int{1, 0}
+		}
+		if votes_layout[j][0] <= votes_layout[j][1] {
+			results_of_all_islands[j] = []int{0, 1}
+		}
+		j++
+		if j > 6 {
+			break
+		}
+	}
```

Comment:
use the `for i := 1; i <= 6; i++` syntax? It's clearer

Reply:
This should also be done throughout the PR